### ISSUE TITLE
all: unset strips line breaks from <pre>

### ISSUE THREAD

**Issue by viktorix** [elad2412/the-new-css-reset#54]
I noticed code blocks using `<pre>` were missing line breaks. The problem seems to stem from this line:
```
*:where(:not(html, iframe, canvas, img, svg, video, audio):not(svg *, symbol *)) {
    all: unset;
    display: revert;
}
```
If I disable `all: unset` in devtools, line breaks return to `<pre>`.  But for the life of me, I can't seem to figure out which CSS property it unsets to remove line breaks so I can add them back.

Here's an example with `all: unset` and while its disabled:
![image](https://user-images.githubusercontent.com/1692600/206272265-ebf6383a-c383-4a20-b5eb-331a63722c6c.png)

Thanks.

**Comment by graymatter00** [comment]
It might be the CSS `white-space` property.

See [https://developer.mozilla.org/en-US/docs/Web/CSS/white-space](https://developer.mozilla.org/en-US/docs/Web/CSS/white-space).

**Comment by elad2412** [comment]
Added fix in the new version (1.8.0) revert for <pre> and <modal> element
